Tools/AP_Periph/wscript

```diff
@@ -37,6 +37,10 @@ def build(bld):
         'AP_RangeFinder',
         'AP_ROMFS',
         'AP_MSP',
+        'SRV_Channel',
+        'AP_Volz_Protocol',
```

Comment:
we could ifdef for AP_Periph to avoid this

Reply:
removed